### PR TITLE
Assignment 5

### PR DESCRIPTION
**Pull request by cadencjk**

### CODE REVIEW

File: Collaboration/.gitignore
Comment (lhy-hoyin) [inline Collaboration/.gitignore:1]: EOF new line for consistency

**Review by JrmCkh — APPROVED** [review]
LGTM

File: template.env
Comment (lhy-hoyin) [inline template.env:1]: EOF new line for consistency

File: frontend/src/css/QuestionContent.css
Comment (lhy-hoyin) [inline frontend/src/css/QuestionContent.css:1]: EOF new line for consistency

File: frontend/src/css/OverlayCursor.css
Comment (lhy-hoyin) [inline frontend/src/css/OverlayCursor.css:1]: EOF new line for consistency

File: frontend/src/css/Modal.css
Comment (lhy-hoyin) [inline frontend/src/css/Modal.css:1]: EOF new line for consistency

File: frontend/src/css/CountdownTimer.css
Comment (lhy-hoyin) [inline frontend/src/css/CountdownTimer.css:1]: EOF new line for consistency

File: frontend/src/css/Chat.css
Comment (lhy-hoyin) [inline frontend/src/css/Chat.css:1]: EOF new line for consistency

File: frontend/src/pages/Signup.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React, { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
-import { signup } from '../api/UserApi.js';
-import { showSuccessToast } from '../utils/toast.js';
-import { errorHandler } from '../utils/errors.js';
+import { signup } from '../api/UserApi'
```
Comment (lhy-hoyin) [inline frontend/src/pages/Signup.js:3]: Missing ending `;` (for consistency)

File: docs/LocalRabbitMqSetup.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Setup RabbitMQ Locally
+
+## Install Erlang on Computer
+
+- Download and install [Erlang](https://www.erlang.org/downloads) before you install the
+  [RabbitMQ](https://www.rabbitmq.com/download.html) server
+- For Windows OS, it is recommended to install the 64-bit version
```
Comment (lhy-hoyin) [inline docs/LocalRabbitMqSetup.md:7]: Which version they install depends on their machine. I think its automatically understood to install the 64-bit version my the machine is 64-bits

File: Nginx/nginx.conf
Comment (lhy-hoyin) [inline Nginx/nginx.conf:1]: Caution: The services name are hardcoded, meaning that if  `compose.yml` is changed, this will break.

File: Match/.gitignore
Comment (lhy-hoyin) [inline Match/.gitignore:1]: EOF newline for consistency

**Review by lhy-hoyin — COMMENTED** [review]
Some minor changes

**Pull request merged**